actors/builtin/reward/reward_actor.go

```diff
@@ -112,6 +113,17 @@ func (a Actor) ThisEpochReward(rt vmr.Runtime, _ *adt.EmptyValue) *abi.TokenAmou
 	return &st.ThisEpochReward
 }
 
+// The baseline power the network is targeting this epoch, updated at the end of an
+// epoch through cron tick.  In the case previous epochs were null blocks this is
+// the value as calculated at the last non-null epoch
+func (a Actor) ThisEpochBaselinePower(rt vmr.Runtime, _ *adt.EmptyValue) *abi.StoragePower {
```

Comment:
Maybe we want to just group this return in with `ThisEpochReward`'s return and we could rename it to something more general like `ThisEpochStats`